app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,20 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(
+            self,
+            hummer_level: int,
+            favourite_dish: str,
+            nickname: str
+    ) -> None:
+
+        super().__init__(favourite_dish, nickname)
+        self._hummer_level = hummer_level
+
+    def get_rating(self) -> int:
+        return 4 + self._hummer_level
+
+    def player_info(self) -> str:
+        return (f"Dwarf warrior {self.nickname}. {self.nickname} has a hummer "

```

Comment:
Format for better readability
```suggestion
        return (
            f"Dwarf warrior {self.nickname}. {self.nickname} has a hummer "

```